src/annotator/config/test/index-test.js

```diff
@@ -1,25 +1,22 @@
 'use strict';
 
 var proxyquire = require('proxyquire');
+var util = require('../../../shared/test/util');
 
 var fakeSharedSettings = {
   jsonConfigsFrom: sinon.stub(),
 };
 var fakeSettings = {};
 
-var configFrom = proxyquire('../index', {
+var configFrom = proxyquire('../index', util.noCallThru({
   './settings': fakeSettings,
   '../../shared/settings': fakeSharedSettings,
-});
+}));
 var sandbox = sinon.sandbox.create();
 
-function fakeWindow(someHref) {
-  var href = someHref || 'LINK_HREF';
-
+function fakeWindow() {
   return {
-    document: {
-      querySelector: sinon.stub().returns({href: href}),
-    },
+    document: 'THE_DOCUMENT',
```

Comment:
In the `index.js` tests we no longer need to mock the DOM (`querySelector` etc). In the `settings.js` unit tests for any particular settings functions that do use the DOM, we can use the real DOM just for those functions own unit tests.